scripts/train/run_batch.sh

```diff
@@ -67,13 +96,14 @@ export SHOW_PROXY_URL=no
 export LABELS="{gcpllm=\"$CLUSTER_PREFIX\"}"
 export MINIMIZE_TERRAFORM_LOGGING=true
 #export DISK_SIZE_GB=1000
-/usr/entrypoint.sh
+_invoke_cluster_tool
 
 echo "Provishioning cluster..."
 
 (sleep 2400;echo check > check.txt) &
 
 
+monitoring_started=false
```

Comment:
To be consistent with the rest of the fine. Let's make this capital case

Reply:
I believe others are environment variables. 
https://google.github.io/styleguide/shellguide.html#variable-names